UPGRADE.md

```diff
@@ -1,3 +1,39 @@
+# Ilios 3 Upgrade Notes
+
+## General steps
+
+0. Back up your database. _Always._
+
+1. Upgrade you codebase via composer.
+
```

Comment:
I don't know if it is always required, but I've had the best luck with upgrading by also nuking vendor/* before running composer installs.  In my experience, items left in this folder during an upgrade has caused problems in the past.